src/components/Feed/feed.css

```diff
@@ -1,6 +1,6 @@
 .feed-container{
   flex: 50%;
-  height: 90vh;
+  /* height: 90vh; */
 }
 
 .feed-wrapper{
```

Comment:
give a max-width to this wrapper.
In larger screen it becomes very big
Attaching a screenshot for your reference
![abhishek](https://user-images.githubusercontent.com/95222845/168817258-41659afc-66c3-4b04-9db9-f1aa17ca1e12.jpg)

Reply:
fixed